### PR TITLE
config: Make default replica check quickly.

### PR DESCRIPTION
**Pull request by ngaut**
Tweak log message.

The change would make recovery testing easy.

### CODE REVIEW

File: src/raftstore/store/peer.rs [hunk 1]
```diff
@@ -716,9 +716,9 @@ impl Peer {
                 self.peer_cache.wl().insert(peer.get_id(), peer.clone());
                 region.mut_peers().push(peer.clone());
 
-                warn!("my peer id {}, add peer {}, region {:?}",
+                warn!("my peer id {}, add peer {:?}, region {:?}",
                       self.peer_id(),
```
Comment (siddontang) [inline src/raftstore/store/peer.rs]: /peer_id/id/

File: src/raftstore/store/peer.rs [hunk 2]
```diff
@@ -685,10 +685,10 @@ impl Peer {
         let change_type = request.get_change_type();
         let mut region = self.region();
 
-        warn!("my peer id {}, {:?} {}, epoch: {:?}\n",
+        warn!("my peer id {}, {}, {:?}, epoch: {:?}\n",
+              self.peer.get_id(),
```
Comment (siddontang) [inline src/raftstore/store/peer.rs]: use id()

**Comment by ngaut** [comment]
PTAL

**Comment by siddontang** [comment]
LGTM

**Pull request merged**